files/en-us/web/javascript/reference/global_objects/regexp/exec/index.html

```diff
@@ -184,6 +184,9 @@ <h3 id="Finding_successive_matches">Finding successive matches</h3>
 
   <p>Also, be sure that the global flag ("<code>g</code>") is set, or it will also cause
     an infinite loop.</p>
+
+  <p>In addition, when matching zero-length characters (e.g. <code>/^/gm</code>),
+    <code>++regexp.lastIndex</code> can be used to avoid infinite loops.</p>
```

Comment:
```suggestion
    {{jsxref("RegExp.lastIndex", "lastIndex")}} can be used to avoid infinite loops.</p>
```

Looks great @keePeek thank you! Just a minor update to include a link to `lastIndex` 👍